### DISCUSSION TITLE
Display test run errors in more readable form.

### DISCUSSION THREAD

**Discussion by PawelWozniak** [forcedotcom/salesforcedx-vscode#5831] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, when I run a test in VSCode and an error happens I need to scroll down through output as the whole message is placed in a narrow column. It is even worse visible if you have a mini panel opened only. Image below from maximized panel.

![2021-11-15_10h15_05](https://user-images.githubusercontent.com/5740564/141769129-6e09176d-9260-4a28-bf7b-d32f2bc244de.png)

**Describe the solution you'd like**
Text is displayed in a full row.

**Describe alternatives you've considered**
As a future rich alternative, an error is displayed in code over the line reported as faulty.

**Additional context**

**Comment by AnanyaJha** [comment]
Hi @PawelWozniak thanks for sharing this feedback! Out of curiosity, what do the tests results look like for you when you run them directly in the terminal/CLI? Does this issue ever occur there, or is it specific to VS Code and its output tab?
